support-workers/src/main/scala/com/gu/emailservices/EmailFields.scala

```diff
@@ -4,15 +4,26 @@ import com.gu.i18n.Currency
 import com.gu.salesforce.Salesforce.SfContactId
 import com.gu.support.promotions.Promotion
 import com.gu.support.workers.{BillingPeriod, User}
-import io.circe.Encoder
+import io.circe.{Encoder, Printer}
 import io.circe.generic.semiauto._
 import io.circe.syntax._
+import org.joda.time.{DateTime, DateTimeZone, LocalDate, LocalTime}
+import com.gu.support.encoding.CustomCodecs._
+import org.joda.time.format.ISODateTimeFormat
 
 case class EmailPayloadContactAttributes(SubscriberAttributes: Map[String, String])
 case class EmailPayloadTo(Address: String, ContactAttributes: EmailPayloadContactAttributes)
-case class EmailPayload(To: EmailPayloadTo, DataExtensionName: String, SfContactId: Option[String], IdentityUserId: Option[String])
+case class EmailPayload(
+  To: EmailPayloadTo,
+  DataExtensionName: String,
+  SfContactId: Option[String], // TODO delete this and make IdentityId non Option
```

Comment:
this was a request made by paul brown, that us using SF contact causes difficulties and that we should use identity id where possible (which is all cases for support-workers)